etc/conf.d/server.lua

```diff
@@ -38,6 +38,6 @@ opts = {
   iat_slack = 600,
   redirect_uri_scheme = "https",
   logout_path = "/logout",
-  redirect_after_logout_uri = "https://sso.mozilla.com/logout",
+  redirect_after_logout_uri = "https://sso.mozilla.com/forbidden",
```

Comment:
i wonder if we should default to something else, or create a log out page or something like that

Reply:
a log out page would be good but i think this default is fine for now